### PR TITLE
[FLINK-23615][docs] Fix the mistake in "systemfunctions"

### PR DESCRIPTION
**Pull request by Launche**
## What is the purpose of the change
Fix the mistake in "systemfunctions".

The documentation of the TRUNCATE SQL function states that "E.g. 42.324.truncate(2) to 42.34", which should be "42.32".
Some period in the document also lacks trailing spaces.

## Brief change log

  - The description of TRUNCATE SQL function has been modified in docs/data/sql_functions.yml and docs/data/sql_functions_zh.yml
  - Add spaces  in docs/data/sql_functions_zh.yml where the specification requires 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): ( no )
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no )
  - The serializers: ( no )
  - The runtime per-record code paths (performance sensitive): ( no )
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: ( no )
  - The S3 file system connector: ( no )

## Documentation

  - Does this pull request introduce a new feature? ( no )
  - If yes, how is the feature documented? ( no )

### CODE REVIEW

File: docs/data/sql_functions.yml [hunk 1]
```diff
@@ -240,7 +240,7 @@ arithmetic:
...
   - sql: TRUNCATE(numeric1, integer2)
     table: numeric1.truncate(INTEGER2)
-    description: Returns a numeric of truncated to integer2 decimal places. Returns NULL if numeric1 or integer2 is NULL.If integer2 is 0,the result has no decimal point or fractional part.integer2 can be negative to cause integer2 digits left of the decimal point of the value to become zero.This function can also pass in only one numeric1 parameter and not set Integer2 to use.If Integer2 is not set, the function truncates as if Integer2 were 0. E.g. 42.324.truncate(2) to 42.34. and 42.324.truncate() to 42.0.
+    description: Returns a numeric of truncated to integer2 decimal places. Returns NULL if numeric1 or integer2 is NULL.If integer2 is 0,the result has no decimal point or fractional part.integer2 can be negative to cause integer2 digits left of the decimal point of the value to become zero.This function can also pass in only one numeric1 parameter and not set Integer2 to use.If Integer2 is not set, the function truncates as if Integer2 were 0. E.g. 42.324.truncate(2) to 42.32. and 42.324.truncate() to 42.0.
```
Comment (tsreaper) [inline docs/data/sql_functions.yml:243]: There should be a space after a comma or a period. For example "integer2 is NULL.If integer2 is 0,the" should be "integer2 is NULL. If integer2 is 0, the". Please check this paragraph again and fix them.

**Review by tsreaper — COMMENTED** [review]
Thanks for your interest in contributing to Flink! I've left a comment.

File: docs/data/sql_functions.yml (hunk 1 above)
Comment (Launche) [reply docs/data/sql_functions.yml:243]: Thanks for your advice.  I ignored the fact that this approach is common sense in the document specification, although I did not find the entry in the Documentation Style Guide. I will fix these issues immediately

**Comment by Launche** [comment]
I have observed that the case of data types in the document is not uniform.  Some words in the description are lowercase, the first letter of some words is capitalized, and some words are capitalized.
For example "integer2 可以为负数，使值的小数点左边的 integer2 位变为零。此函数也可以传入只有一个 numeric1 参数且不设置 Integer2 以使用。如果未设置 Integer2 则 Integer2 为 0。" and "将字符串 STRING1 按照 STRING2 正则表达式的规则拆分，返回指定 INTEGER1 处位置的字符串。正则表达式匹配组索引从 1 开始"

**Comment by Launche** [comment]
Does the style need to be unified？

**Comment by tsreaper** [comment]
> Does the style need to be unified？

From my point of view it will be great to unify this if the number of changes is limited. If this change will touch a lot of places then I suggest leaving it as it is.

**Comment by Launche** [comment]
I quite agree with you. I had a quick look at some of the documentation, and this could be quite a project. If necessary I would suggest that a new issue be created specifically to deal with this phenomenon.

**Comment by Launche** [comment]
> > Does the style need to be unified？
> 
> From my point of view it will be great to unify this if the number of changes is limited. If this change will touch a lot of places then I suggest leaving it as it is.

I quite agree with you. I had a quick look at some of the documentation, and this could be quite a project. If necessary I would suggest that a new issue be created specifically to deal with this phenomenon.

**Review by tsreaper — APPROVED** [review]
Looks good to me now. @wuchong any other suggestions?

**Comment by wuchong** [comment]
Thanks @tsreaper for the reviewing and @Launche for the contribution. 
Merging...

**Comment by Launche** [comment]
Thanks a lot. This is a memorable contribution for me.

**Pull request merged**